aspnetcore/fundamentals/host/hosted-services/samples/8.0/BackgroundTasksSample/BackgroundTasksSample.csproj

```
@@ -0,0 +1,14 @@
+<Project Sdk="Microsoft.NET.Sdk.Worker">
+
+	<PropertyGroup>
+		<TargetFramework>net8.0</TargetFramework>
+		<UserSecretsId>dotnet-BackgroundTasksSample-E6370564-EEA4-467C-AC17-5BDD716D3C4B</UserSecretsId>
+		<RootNamespace>BackgroundTasksSample</RootNamespace>
```

Delete unnecessary GUID

```suggestion
		<TargetFramework>net8.0</TargetFramework>
		<RootNamespace>BackgroundTasksSample</RootNamespace>
```